src/test/java/io/confluent/copycat/hdfs/HdfsSinkConnectorTestBase.java

```diff
@@ -99,15 +115,55 @@ protected GenericRecord createRecord() {
   @Before
   public void setUp() throws IOException {
     conf = new Configuration();
-    cluster = createDFSCluster(conf);
-    url = "hdfs://" + cluster.getNameNode().getClientNamenodeAddress();
-    fs = cluster.getFileSystem();
+    conf.setBoolean("dfs.client.block.write.replace-datanode-on-failure.enable", true);
```

Comment:
What do these two lines accomplish? It looks like you're enabling a setting, then adjusting another so it is never used.